### ISSUE TITLE
[Finetune] Should I reduce `cfg.max_steps` when using more GPUs?

### ISSUE THREAD

**Issue by KaijingOfficial** [openvla/openvla#149]
Under the setting of DDP, when I use more GPU. I have to reduce the batch_size to maintain a fixed global batch size. However, when I reduce `batch_size`, there's a lot GPU memory wasted.  So I'm thinking keep the batch size while reduce the `cfg.max_steps`.  Can I get same or similar results?

```
    # Train!
    with tqdm.tqdm(total=cfg.max_steps, leave=False) as progress:
           ...
```

**Comment by moojink** [comment]
Hi @KaijingOfficial,

If you use more GPUs, I would recommend that you keep the batch size the same as before. You will have a larger total batch size, and you should observe faster training convergence with better gradients. Yes, you should be able to reduce `cfg.max_steps` given faster training, though it may be hard to predict how much faster training will convergence.

-Moo Jin
